drivers/bridge/bridge.go

```diff
@@ -468,35 +468,6 @@ func (d *driver) CreateNetwork(id types.UUID, option map[string]interface{}) err
 	bridgeIface := newInterface(config)
 	network.bridge = bridgeIface
 
-	// Verify network does not conflict with previously configured networks
```

Comment:
You cannot remove this section without replacing it with something equivalent: As the comment says, at this stage we need to run the conflict check again because new fields might have been added to the config structure, during the setup steps.